### PR TITLE
updates to grid-related content

### PR DESCRIPTION
**Pull request by topepo**
- Move to `grid_space_filling()` 
- Show a slide for different designs
- Added a screen shot about racing from Mike M

### CODE REVIEW

**Review by EmilHvitfeldt — APPROVED** [review]

File: slides/advanced-03-tuning-hyperparameters.qmd [hunk 1]
```diff
@@ -155,18 +155,6 @@ These are based on prior belief.
...
-:::
-
-::: fragment
-It will impact inference though.
-🤔
-:::
-
-
 
 # Is the random seed a tuning parameter?
```
Comment (simonpcouch) [inline slides/advanced-03-tuning-hyperparameters.qmd:159]:
```suggestion
# The random seed?
```

If this parallels the phrasing of the slides around it:

**Review by simonpcouch — APPROVED** [review]
Looking snazzy🏄

File: slides/intro-01-introduction.qmd [hunk 2]
```diff
@@ -58,7 +58,7 @@ Wi-Fi password
 
 ## Workshop policies
 
--   Please review the code of conduct: <https://latin-r.com/en/sobre/coc/code-of-conduct.html>
+-   Please review the code of conduct: <https://github.com/tidymodels/workshops/blob/main/CODE_OF_CONDUCT.md>
```
[the remark this answers is not in the document]
Comment (topepo) [reply slides/intro-01-introduction.qmd:61]: I'll leave as-is then update form main

File: slides/advanced-03-tuning-hyperparameters.qmd
[the remark this answers is not in the document]
Comment (topepo) [reply slides/advanced-03-tuning-hyperparameters.qmd:1]: Yeah. I fix that later in the other PR

**Pull request merged**